packages/pebble-web/src/components/shared/Control.tsx

```diff
@@ -52,10 +52,10 @@ export const ControlView: React.FunctionComponent<ControlViewProps> = ({
   // Ensure that other styles are not emotion styles.
   // As cx merges styles into one className.
   const iconClass = cx(radioIconStyle, "pi", {
-    "pi-radio": isRadio && !checked && !disabled,
-    "pi-radio-selected": isRadio && (checked || disabled),
-    "pi-checkbox-selected": !isRadio && (checked || disabled),
-    "pi-checkbox-unselected": !isRadio && !checked && !disabled
+    "pi-radio": isRadio && !checked,
+    "pi-radio-selected": isRadio && checked,
+    "pi-checkbox-selected": !isRadio && checked,
+    "pi-checkbox-unselected": !isRadio && !checked
```

Comment:
Why remove checks for disabled?

Reply:
This was supposed to go in different PR.
This is because if we were disabling the checkbox or radio, it was showing disabled selected.